Декоративные элементы на главной странице

v1aver opened the pull request:
---
:mortar_board: [Декоративные элементы на главной странице](https://up.htmlacademy.ru/htmlcss/33/user/1813431/tasks/10)

v1aver committed:
Декоративные элементы на главной и внутренней страницах

v1aver committed:
Update styles.css

brewscriptor commented on styles/styles.css:
```diff
@@ -573,35 +824,56 @@ body {
+  /* width: 55px;
+  height: 55px; */
+  border-radius: 4px;
+  box-sizing: border-box;
```
это правило лучше задавать сверху в начале проекта, чтобы везде не писать

brewscriptor commented on catalog.html:
```diff
@@ -18,29 +18,36 @@
                             <span class="visually-hidden">Поиск</span>
-                            <img class="icon-search" src="/images/icon/icon-search.svg" width="18" height="18"
-                                alt="Иконка поиска">
+                            <svg class="navigation-icon" aria-hidden="true" focusable="false" width="20" height="19" viewBox="0 0 20 19"
```
нужно везде удалить focusable="false"

brewscriptor commented on catalog.html:
```diff
@@ -146,17 +153,36 @@ <h2 class="visually-hidden">Список отелей с фильтрами</h2>
 
                     <div class="mode-container">
-                        <a class="button-mode button-mode-selected" href="#">
+                        <a class="button-mode button-mode-selected" href="#" type="button">
```
у ссылок а нужно везде удалить type="button"

v1aver committed:
Правки

- Добавляет в html box-sizing.
- Удаляет focusable="false" у svg.
- Удаляет type="button" у ссылок а.
